docs/ruleset_functions/get_zone_eflh.md

```diff
@@ -0,0 +1,43 @@
+# get_zone_eflh
+
+**Description:** provides the equivalent full load hours of the zone.  Equivalent full load hours are defined as: any hour where the occupancy fraction is greater than 5% AND the HVAC system is in occupied mode.  For this function, we are recognizing the HVAC system as being in occupied mode if ANY of the HVAC systems serving the zone are in occupied mode.
+**Applicability Note:** hvac_system.fan_system.operation_schedule is being used as the HVAC operation schedule.  Therefore, this check will not work for radiant systems or other systems that do not include a fan
+
+**Inputs:** 
+- **RMD**
+- **zone**
+
+**Returns:**  
+- **flh**: a number equal to the total equivalent full load hours for the year
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- create a list to hold information about the occupancy of each space: `people_info_for_spaces = {}`
+- create a variable for the total number of zone occupants: `total_zone_occupants = 0`
+- loop through the spaces in the zone collecting information about occupancy: `for space in zone.spaces:`
+  - append the information about the occupancy to the list: `people_info_for_spaces[space]["NUM_OCCUPANTS] = space.number_of_occupants`
+  - `people_info_for_spaces[space]["SCHEDULE_HOURLY_VALUES"] = space.occupant_multiplier_schedule.hourly_values`
+  - add the occupants of this space to the total number of zone occupants: `total_zone_occupants += space.number_of_occupants`
+- get the list of the hvac systems associated with the zone: `hvac_systems_list = get_list_hvac_systems_associated_with_zone(RMD,zone.id)`
+- create an index to which each full load hour will be added: `flh = 0`
+- now create a variable num_hours_in_year which will take the number of hours in the schedule for the first (really any) space in people_info_for_spaces: `num_hours_in_year = len(people_info_for_spaces[list(people_info_for_spaces.keys())[0]]["SCHEDULE_HOURLY_VALUES"])`
```

Comment:
Should we include a check here?

```python
if not all(len(people_info_for_spaces[key]["schedule_hourly_values"]) == num_hours_in_year for key in people_info_for_spaces)
raise exception "Hourly schedules have different number of hours in a year"
```
This might be either addressed in the umbrella check or address in here.

Reply:
> Should we include a check here?
> 
> ```python
> if not all(len(people_info_for_spaces[key]["schedule_hourly_values"]) == num_hours_in_year for key in people_info_for_spaces)
> raise exception "Hourly schedules have different number of hours in a year"
> ```
> 
> This might be either addressed in the umbrella check or address in here.

@mkarpman - thoughts?

@weilixu - can you explain how exceptions are handled?  Does this simply give the user an alert, or does it stop the RCT from completing the check?

Reply:
Seems like this should be part of RMD validity check when the file is parsed by RCT. If ScheduleSequenceOption=hourly, but the schedule has fewer than 8760 values, the RMD should be rejected with a report listing all areas where RMD violates the spec. .